psi4/src/psi4/libmints/local.cc

```diff
@@ -549,4 +550,544 @@ void PMLocalizer::localize() {
     U_->transpose_this();
 }
 
+IBOLocalizer::IBOLocalizer(std::shared_ptr<BasisSet> primary, std::shared_ptr<BasisSet> minao,
+                             std::shared_ptr<Matrix> C)
+    : Localizer(primary, C), minao_(minao) {
+    common_init();
+}
+IBOLocalizer::~IBOLocalizer() {}
+void IBOLocalizer::common_init() {
```

Comment:
`common_init` is not marked `override`. Is it meant to override the base class constructor or not?